Tasks/Common/VstsAzureHelpers_/InitializeAzureRMFunctions.ps1

```diff
@@ -120,33 +118,33 @@ function Initialize-AzureRMSubscription {
                 if (CmdletHasMember -cmdlet "Add-AzureRMAccount" -memberName "EnvironmentName") {
                         
                     if ($Endpoint.Auth.Parameters.AuthenticationType -eq "SPNCertificate") {
-                        Write-Host "##[command]Add-AzureRMAccount -ServicePrincipal -Tenant $($Endpoint.Auth.Parameters.TenantId) -CertificateThumbprint ****** -ApplicationId $($Endpoint.Auth.Parameters.ServicePrincipalId) -EnvironmentName $environmentName"
-                        $null = Add-AzureRmAccount -ServicePrincipal -Tenant $Endpoint.Auth.Parameters.TenantId -CertificateThumbprint $servicePrincipalCertificate.Thumbprint -ApplicationId $Endpoint.Auth.Parameters.ServicePrincipalId -EnvironmentName $environmentName
+                        Write-Host "##[command]Add-AzureRMAccount -ServicePrincipal -Tenant $($Endpoint.Auth.Parameters.TenantId) -CertificateThumbprint ****** -ApplicationId $($Endpoint.Auth.Parameters.ServicePrincipalId) -EnvironmentName $environmentName -Scope Process"
```

Comment:
ensure you test this with both type of agent config. - interactive and runAsService

Reply:
thanks, Chaitanya. verified.